test/utils/operator/orderSurplus.test.ts

```diff
@@ -15,129 +15,129 @@ describe('getOrderSurplus', () => {
   describe('Buy order', () => {
     describe('fillOrKill', () => {
       test('No surplus', () => {
-        const order: RawOrder = {
+        const order = {
           ...RAW_ORDER,
           kind: 'buy',
           sellAmount: '100',
           executedSellAmount: '100',
           feeAmount: '0',
           executedFeeAmount: '0',
           partiallyFillable: false,
-        }
+        } as RawOrder
         expect(getOrderSurplus(order)).toEqual({ amount: ZERO_BIG_NUMBER, percentage: ZERO_BIG_NUMBER })
       })
       test('No matches', () => {
-        const order: RawOrder = {
+        const order = {
           ...RAW_ORDER,
           kind: 'buy',
           sellAmount: '100',
           executedSellAmount: '0',
           feeAmount: '0',
           executedFeeAmount: '0',
           partiallyFillable: false,
-        }
+        } as RawOrder
         expect(getOrderSurplus(order)).toEqual({ amount: ZERO_BIG_NUMBER, percentage: ZERO_BIG_NUMBER })
       })
       test('With fees = 0', () => {
-        const order: RawOrder = {
+        const order = {
           ...RAW_ORDER,
           kind: 'buy',
           sellAmount: '100',
           executedSellAmount: '99',
           feeAmount: '0',
           executedFeeAmount: '0',
           partiallyFillable: false,
-        }
+        } as RawOrder
         expect(getOrderSurplus(order)).toEqual({ amount: ONE_BIG_NUMBER, percentage: ZERO_DOT_ZERO_ONE })
       })
       test('With fees > 0', () => {
-        const order: RawOrder = {
+        const order = {
           ...RAW_ORDER,
           kind: 'buy',
           sellAmount: '100',
           executedSellAmount: '109', // 10 is the fee, total sold is 99; surplus === 1
           feeAmount: '10',
           executedFeeAmount: '10',
           partiallyFillable: false,
-        }
+        } as RawOrder
         expect(getOrderSurplus(order)).toEqual({ amount: ONE_BIG_NUMBER, percentage: ZERO_DOT_ZERO_ONE })
       })
     })
     test.skip('partiallyFillable', () => {
-      const order: RawOrder = {
+      const order = {
         ...RAW_ORDER,
         kind: 'buy',
         sellAmount: '100',
         executedSellAmount: '50',
         buyAmount: '100',
         executedBuyAmount: '40',
         partiallyFillable: true,
-      }
+      } as RawOrder
       expect(getOrderSurplus(order)).toEqual({ amount: TEN_BIG_NUMBER, percentage: TWENTY_PERCENT })
     })
   })
 
   describe('Sell order', () => {
     describe('fillOrKill', () => {
       test('No surplus', () => {
-        const order: RawOrder = {
+        const order = {
```

Comment:
what was the error that was throwing this?

Reply:
I got some types compatibilites issues. 

```
Type '{ kind: "sell"; buyAmount: string; executedBuyAmount: string; feeAmount: string; executedFeeAmount: string; partiallyFillable: false; creationDate: string; owner: string; receiver?: string | undefined; ... 9 more ...; status: RawOrderStatusFromAPI; } | { ...; }' is not assignable to type 'RawOrder'.
  Type '{ kind: "sell"; buyAmount: string; executedBuyAmount: string; feeAmount: string; executedFeeAmount: string; partiallyFillable: false; creationDate: string; owner: string; uid: string; availableBalance: string; ... 11 more ...; receiver: string; }' is not assignable to type 'RawOrder'.
    Type '{ kind: "sell"; buyAmount: string; executedBuyAmount: string; feeAmount: string; executedFeeAmount: string; partiallyFillable: false; creationDate: string; owner: string; uid: string; availableBalance: string; ... 11 more ...; receiver: string; }' is not assignable to type 'OrderMetaData'.
      Types of property 'kind' are incompatible.
        Type '"sell"' is not assignable to type 'OrderKind'.
```